conf/multiconfig/README.txt

```diff
@@ -0,0 +1,35 @@
+How to to set a multiconfig environment for ledge.
+
+1. create you build environment:
+--------------------------------
+   $> MACHINE=ledge-qemuarm DISTRO=rpb source ./setup-environment build-rpb-mc
+NOTE: for this step the machine name have not relevent.
+
+2. Request a multiconfig build for armhf
+----------------------------------------
+	$> ./conf/multiconfig/launch_bitbake_multi_armhf.sh
+	or
+	$> export BBMULTICONFIG="qemuarm stm32mp157c-dk2 ti-am572x"
+	$> sed -e "s%^BBMULTICONFIG ?= .*$%BBMULTICONFIG ?= \"${MULTICONFIG}\"%" -i conf/auto.conf
+	$> MACHINE=ledge-qemuarm bitbake \
+		mc:qemuarm:ledge-gateway \
+		mc:qemuarm:ledge-iot \
```

Comment:
ledge-qemuarm in both cases.

Reply:
It's normal because you need to specify a machine the during the creation of build environment et after it's just a reminder and on all the bitbake command we preconize for ledge we specify the machine. The advantage are that we can change the machine when we would like to make a build a a specific machine

Reply:
then maybe name it as 'qemu'? I don't like transision here qemu-arm to ledge-qemuarm. If it's the same naming it's more easy to follow code.

Reply:
It's just to have e different naming between configuration name and machine name.